.github/workflows/e2e-test.yml

```
@@ -109,17 +121,21 @@ jobs:
+          deploy_pypi_server: ${{ matrix.deploy_pypi_server }}ic
+          proxy: ${{ matrix.proxy }}
           pod_to_pod_tls_enabled: ${{ matrix.pod_to_pod_tls_enabled }}
           image_path: ${{ needs.build.outputs.IMAGE_PATH }}
           image_tag: ${{ needs.build.outputs.IMAGE_TAG }}
           image_registry: ${{ needs.build.outputs.IMAGE_REGISTRY }}
 
-      - name: Configure Cluster CA Cert for TLS-Enabled Tests
+      - name: Configure Cluster CA e2e for TLS-Enabled Tests
         shell: bash
         if: ${{ matrix.pod_to_pod_tls_enabled == 'true'}}
         run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:2:55: Double quote to prevent globbing and word splitting

(shellcheck)